lib/buildsystems/perl.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class PErl < Package
+  property :pre_perl_options
+
+  def self.prebuild
+    puts "Additional pre_perl_options being used: #{@pre_perl_options.nil? ? '<no pre_perl_options>' : @pre_perl_options}".orange
+    system "#{@pre_perl_options} perl Makefile.PL"
+    system "sed -i 's,/usr/local,#{CREW_PREFIX},g' Makefile"
```

Comment:
Could this be in `def self.patch` or does that run before prebuild?

Reply:
patch runs before prebuild